src/R/Components/Impl/PackageManager/Implementation/RPackageManager.cs

```diff
@@ -36,11 +52,11 @@ internal class RPackageManager : IRPackageManager {
         }
 
         public async Task<IReadOnlyList<RPackage>> GetInstalledPackagesAsync() {
-            return await GetPackages(async (eval) => await eval.InstalledPackages());
+            return await GetPackagesAsync(async (eval) => await eval.InstalledPackages());
```

Comment:
I'd like to be able to execute `GetInstalledPackagesAsync` and `GetAvailablePackagesAsync` concurrently. `GetAvailablePackagesAsync` is much slower, while `GetInstalledPackagesAsync` will be executed every time user switch the tab